spec/features/users_index_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.feature 'Logins', type: :feature do
+  background do
+    visit root_path
+  end
+
+  scenario 'show groups_path' do
+    expect(page).to have_content('Log in')
+  end
+
+  scenario 'show groups_path' do
+    expect(page).to have_content('Sign up')
+  end
+end
```

Comment:
- [ ] Some of your tests are failing, Kindly fix this
 
![Screenshot from 2022-08-05 21-58-34](https://user-images.githubusercontent.com/66097371/183169976-ba9a7202-0eee-4e35-933d-7f34a0332587.png)

Reply:
Thanks man for the feedback,I fixed that, and below is the screenshot

![image](https://user-images.githubusercontent.com/81693905/183204262-6354784c-523a-4358-941c-755af72ca0c4.png)